runtime/onert/frontend/base_loader/include/base_loader.h

```diff
@@ -243,6 +245,35 @@ void BaseLoader<LoaderDomain>::BaseLoader::loadFromBuffer(uint8_t *buffer, size_
   loadModel();
 }
 
+template <typename LoaderDomain>
+std::unique_ptr<ir::Data>
+BaseLoader<LoaderDomain>::BaseLoader::bufferIndexToData(const uint32_t buffer_idx)
+{
+  if (_domain_model == nullptr)
+  {
+    throw std::runtime_error{"fail to access _domain_model"};
+  }
+
+  const auto *data = _domain_model->buffers()->Get(buffer_idx)->data();
```

Comment:
It would be better to check:

- `buffers()` != nullptr
- buffer_idx within range

either runtime or assert.

Please find my check code in https://github.com/Samsung/ONE/pull/12246/files#diff-6062b879f8bbfea9156587d03d744ae19cc190babd74b1afbb0a5fb2d84ab4fb